packages/protocol/contracts/governance/Election.sol

```diff
@@ -851,22 +851,30 @@ contract Election is
     // Holds the number of members elected for each of the eligible validator groups.
     uint256[] memory numMembersElected = new uint256[](electionGroups.length);
     uint256 totalNumMembersElected = 0;
-    // Assign a number of seats to each validator group.
-    while (totalNumMembersElected < maxElectableValidators) {
-      uint256 groupIndex = 0;
-      bool memberElected = false;
-      (groupIndex, memberElected) = dHondt(
-        electionGroups,
-        numMembers,
-        totalNumMembersElected,
-        numMembersElected
-      );
 
-      if (memberElected) {
+    uint256[] memory keys = new uint256[](electionGroups.length);
+    for (uint256 i = 0; i < electionGroups.length; i++) {
+      keys[i] = i;
+    }
+    FixidityLib.Fraction[] memory groupVotes = new FixidityLib.Fraction[](electionGroups.length);
```

Comment:
nit: Consider `votesPerElectedMember`, as `groupVotes` could imply that this stores the group's total vote count